package.json

```diff
@@ -22,35 +26,68 @@
   "license": "MIT",
   "devDependencies": {
     "broccoli": "^0.16.8",
-    "chai": "^1.10.0",
+    "broccoli-asset-rev": "^2.4.2",
+    "chai": "^3.5.0",
+    "chai-files": "^1.4.0",
     "copy-dereference": "^1.0.0",
+    "ember-ajax": "^2.0.1",
+    "ember-cli": "2.8.0",
+    "ember-cli-app-version": "^1.0.0",
+    "ember-cli-dependency-checker": "^1.2.0",
+    "ember-cli-htmlbars": "^1.0.3",
+    "ember-cli-htmlbars-inline-precompile": "^0.3.1",
+    "ember-cli-inject-live-reload": "^1.4.0",
+    "ember-cli-jshint": "^1.0.0",
+    "ember-cli-qunit": "^2.1.0",
+    "ember-cli-release": "^0.2.9",
+    "ember-cli-sri": "^2.1.0",
+    "ember-cli-test-loader": "^1.1.0",
+    "ember-cli-uglify": "^1.2.0",
+    "ember-data": "^2.8.0",
+    "ember-disable-prototype-extensions": "^1.1.0",
+    "ember-export-application-global": "^1.0.5",
+    "ember-load-initializers": "^0.5.1",
+    "ember-resolver": "^2.0.3",
+    "loader.js": "^4.0.1",
     "mocha": "^2.0.1",
-    "mocha-jshint": "0.0.9",
+    "mocha-jshint": "2.3.1",
     "sinon": "^1.12.2",
-    "sinon-chai": "^2.6.0"
+    "sinon-chai": "^2.6.0",
+    "symlink-or-copy": "^1.1.6"
   },
   "dependencies": {
-    "acorn": "^2.6.4",
+    "acorn": "^4.0.0",
     "broccoli-caching-writer": "^3.0.3",
+    "broccoli-funnel": "^1.0.6",
     "broccoli-kitchen-sink-helpers": "^0.3.1",
     "broccoli-merge-trees": "^1.1.2",
     "broccoli-plugin": "^1.2.1",
     "browserify": "^13.0.0",
     "core-object": "^1.1.0",
     "debug": "^2.2.0",
     "derequire": "^2.0.3",
+    "ember-cli-babel": "^5.1.6",
     "ember-cli-version-checker": "^1.1.4",
+    "find-up": "^1.1.2",
     "fs-tree": "^1.0.0",
     "fs-tree-diff": "^0.5.0",
-    "lodash": "^4.5.1",
+    "lodash.merge": "^4.6.0",
     "md5-hex": "^1.3.0",
-    "mkdirp": "^0.5.0",
+    "mkdirp": "^0.5.1",
     "promise-map-series": "^0.2.0",
     "quick-temp": "^0.1.2",
+    "resolve": "^1.1.7",
     "rimraf": "^2.2.8",
     "rsvp": "^3.0.14",
     "symlink-or-copy": "^1.0.0",
     "through2": "^2.0.0",
-    "walk-sync": "^0.2.7"
+    "walk-sync": "^0.3.1"
+  },
+  "ember-addon": {
+    "configPath": "tests/dummy/config",
+    "paths": [
+      "node_modules/modern",
```

Comment:
Because by the time we actually need them in test they're present in node_modules because of the link behavior we use. Probably results in duplication, I need to think about this.